extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/WebSocketServerConnection.java

```diff
@@ -0,0 +1,171 @@
+package io.quarkus.websockets.next;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import io.smallrye.common.annotation.CheckReturnValue;
+import io.smallrye.common.annotation.Experimental;
+import io.smallrye.mutiny.Uni;
+
+/**
+ * This interface represents a client connected to a specific {@link WebSocket} endpoint.
```

Comment:
```suggestion
 * This interface represents a connection from a client to a specific {@link WebSocket} endpoint on the server.
```

seems more precise? Especially given the interface name.

Reply:
Yes, we'll [remove the server part from the name](https://github.com/quarkusio/quarkus/issues/39154) but the argument still applies.